src/django/api/views.py

```diff
@@ -796,9 +798,29 @@ def list(self, request):
             ]
         """
         try:
-            contributor = request.user.contributor
-            queryset = FacilityList.objects.filter(contributor=contributor)
-            response_data = self.serializer_class(queryset, many=True).data
+            if request.user.is_superuser:
```

Comment:
I can't remember if we have tests for these endpoints already, but consider adding tests for each of the codepaths.